tests/testthat/test_binary.R

```diff
@@ -138,8 +138,8 @@ test_that("extra pooling stats work", {
 test_that("Calculation of effects works", {
   expect_is(group_effects(bg5_p), "array")
   expect_is(treatment_effect(bg5_p), "list")
-  expect_length(treatment_effect(bg5_p, summary = TRUE)$tau, 5)
```

Comment:
YOu don't want to overwrite your tests, you want to keep the old tests on `treatment_effect` and then write additional tests to check if `hypermean(bg)` is equal to `treatment_effect(bg, s=T)[[1]]` etc etc. You want to do it for several models/cases, just to be extra sure